storage/client_range_tree_test.go

```diff
@@ -584,10 +592,125 @@ func TestInsertLeft(t *testing.T) {
 		Tree:  tree,
 		Nodes: nodes,
 	}
-	if err := splitRange(db, engine.KeyMin, keyA); err != nil {
+	if err := splitRange(db, keyA); err != nil {
 		t.Fatal(err)
 	}
 	if err := treesEqual(db, expectedTree); err != nil {
 		t.Fatal(err)
 	}
 }
+
+// Compare implements the llrbComparable interface for keys.
+func (k Key) Compare(b llrb.Comparable) int {
+	return bytes.Compare(k, b.(Key))
+}
+
+// nodeSame compares a biogo node and a range tree node to determine if both
+// contain the same values in the same order.  It recursively calls itself on
+// both children if they exist.
+func nodeSame(db *client.KV, biogoNode *llrb.Node, key *proto.Key) error {
```

Comment:
These methods appear to be very similar to the ones for comparing to expected trees at the top of this test file. Why don't you rename this compareBiogoNode for clarity?